types/react/global.d.ts

```diff
@@ -2,6 +2,9 @@
 React projects that don't include the DOM library need these interfaces to compile.
 React Native applications use React, but there is no DOM available. The JavaScript runtime
 is ES6/ES2015 only. These definitions allow such projects to compile with only `--lib ES6`.
+
+Warning: all of these interfaces are empty. If you want type definitions for various properties
+(such as HTMLInputElement.prototype.value), you need to add `--lib DOM` (via command line or tsconfig.json).
```

Comment:
The name of the DOM lib is all lowercase: `typescript/lib/lib.dom.d.ts`

Reply:
This is my source: https://www.typescriptlang.org/docs/handbook/compiler-options.html